scripts/docker/codecompass-build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/env bash
+
+# This script will run inside the Docker container. This will build CodeCompass
+# using the following parameters:
+
+DATABASE=pgsql       # pgsql or sqlite
```

Comment:
How can I configure these parameters when I using this or `start-devenv.sh` scripts?

Reply:
Now you can configure these through environment variables.